lib/stack_master/aws_driver/cloud_formation.rb

```diff
@@ -11,7 +11,9 @@ def delete_stack(options)
       end
 
       def describe_stacks(options)
-        cf.describe_stacks(options)
+        retry_with_backoff do
+          cf.describe_stacks(options)
+        end
       end
```

Comment:
I think not yet.  At least in my experience / usage pattern so far, I haven't had that problem outside of `status` and adding a fix w/o a demonstrated problem is something I like to avoid.

That said, backing up slightly, my workflow / use-case so far has been to do something vaguely like:

```
$ stack_master status | grep -v _COMPLETE | awk '{print $3}' > foo
$ vi foo # trim to taste
$ for i in `cat foo` ; do yes | stack_master apply my-region $i ; done
```

This lets me load a pile of stacks in an unattended fashion.  This does require my `feat/use_gets_instead_of_getch` PR so the `yes` pipe works.

If `apply` accepted a list of stacks, or could work out dependencies among stacks, I think then it would be necessary/useful.  In its current state I don't think its strictly required.  My team and I have discussed building that functionality.